File: src/exemplos/05_EstruturasDeDados/02_Simbolos/01-ascii.py
#  -*- coding: utf-8 -*-
#    @package: 01-ascii.py
#     @author: Guilherme N. Ramos (gnramos@unb.br)
# @disciplina: Algoritmos e Programação de Computadores
#
# Mostra o mapeamento de caracteres ASCII para inteiros.


# Em Python, a codificação determina a representação na memória, mas o valor do
# caractere em ASCII pode ser obtido com a função ord.
for c in 'ABCDEFGHIJKLMNOPQRSTUWXYZ':
    print('%c: %i' % (c, ord(c)))

print()

for c in 'abcdefghijklmnopqrstwxyz':
    print('%c: %i' % (c, ord(c)))

File: src/exemplos/05_EstruturasDeDados/02_Simbolos/01-ascii.c
/**      @file: 01-ascii.c
 *     @author: Guilherme N. Ramos (gnramos@unb.br)
 * @disciplina: Algoritmos e Programação de Computadores
 *
 * Mostra o mapeamento de caracteres ASCII para inteiros. */

#include <stdio.h>

int main() {
    char c;

    for(c = 'A'; c <= 'Z'; ++c)
        printf("%c: %i\n", c, (int) c);

    printf("\n");

    for(c = 'a'; c <= 'z'; ++c)
        printf("%c: %i\n", c, (int) c);

    return 0;
}